README.md

```diff
@@ -2,29 +2,100 @@
+| Register VMs with Red Hat | Uses RHEL subscription manager to register each virtual machine for required RPM repos. |
+| Update VMs | Updates each VM deployed with latest kernel and packages. |
+| Configure SSH on installer VM | Configures the installer VMs with a private SSH key so that it can communicate with the other VMs that are part of the installation process. |
+| Configure RDS Databases | Ensure that the RDS instance has a database for Controller, Hub, and Event-Driven Ansible. |
+| Setup One Controller VM as Installer | Moves the locally downloaded AAP installer to a single VMs and configures the installer inventory file based on the VMs that were created as part of this process. |                 
 
 ## Getting Started
 
-These sections will describe required or recommended steps so that your Ansible Automation Platform deployment is as seamless as possible.
+This section will walk through deploying the AWS infrastructure and Ansible Automation Platform.
 
+You may also download the this repository from GitHub and modify to suit your needs.
 
 ### Red Hat Enterprise Linux
 
 You will need to use a Red Hat Enterprise Linux (RHEL) Amazon Machine Image (AMI) as the foundation for your deployment.  While this collection will automatically find a public RHEL AMI available from AWS, public images bill for RHEL outside of your subscription for Ansible Automation Platform.
 
 It is recommended that you create a custom AMI that you may then use to deploy RHEL with your subscriptions that come with Ansible Automation Platform.  [Red Hat Image Builder][image-builder] is a utility that makes creating a custom AMI easy.
 
+### AWS Credentials
+
+This terraform template requires AWS credentials for deploying infrastructure, which can be set by running command,
+`aws configure`
+
+This template will need a way to connect to the virtual machines that it creates.
+By default, VMs are created with public IP addresses to make this simple, but the template may be modified to use private IP addresses if your local machine can route traffic to private networks.
 
 ## Deploying Ansible Automation Platform
 
 This section will walk through deploying the AWS infrastructure and Ansible Automation Platform.
 
+**NOTE:** This template is designed to provide a quick and easy setup with default values for the number of instances and instance types. However, these values can be customized to meet your specific requirements. 
+
+### Checklist
+
+- [ ] Install this repository
+- [ ] Terraform installed locally (`terraform`)
+- [ ] A RHEL AMI (if not using hourly RHEL instances)
+- [ ] A locally downloaded copy of the [AAP installer][aap-installer]
+- [ ] A variables file configured with required variables
+- [ ] Configure the AWS environment variables for authentication
+- [ ] An inventory file with the proper SSH configuration
+
 ### Deploying Infrastructure
 
+The variables below are required for running this template
```

Comment:
Please add info here on:
infrastructure_controller_ami=""
infrastructure_hub_ami="" 
infrastructure_eda_ami=""
infrastructure_execution_ami="" 
because these arguments are required AND also explain that you can put "" for the value and it will pick a RHEL AMI automatically.  This was not obvious to me and wonder why "" is not the default